app/controllers/application_controller.rb

```diff
@@ -10,6 +10,9 @@ class ApplicationController < ActionController::Base
   )
 
   helper_method :admin_signed_in?, :current_claim, :claim_timeout_in_minutes, :claim_timeout_warning_in_minutes
+
+  before_filter :redirect_www
```

Comment:
I think this should be `before_action`

`before_filter` was removed in Rails 5.1